.github/workflows/sycl-nightly.yml

```diff
@@ -102,6 +102,48 @@ jobs:
             image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
             target_devices: level_zero:gpu
             tests_selector: cts
+
+          - name: Intel L0 GPU w/ LLVM SPIR-V Backend
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
+            target_devices: level_zero:gpu
+            reset_intel_gpu: true
+            tests_selector: e2e
+            extra_lit_opts: --param spirv-backend=True --param gpu-intel-gen12=True
+
+          - name: Intel OCL GPU w/ LLVM SPIR-V Backend
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:gpu
+            reset_intel_gpu: true
+            tests_selector: e2e
+            extra_lit_opts: --param spirv-backend=True --param gpu-intel-gen12=True
+
+          - name: OCL CPU (Intel/GEN12) w/ LLVM SPIR-V Backend
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:cpu
+            tests_selector: e2e
+            extra_lit_opts: --param spirv-backend=True --param gpu-intel-gen12=True
+
+          - name: SYCL-CTS on OCL CPU w/ LLVM SPIR-V Backend
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:cpu
+            tests_selector: cts
+            extra_lit_opts: --param spirv-backend=True --param gpu-intel-gen12=True
+
+          - name: SYCL-CTS on L0 gen12 w/ LLVM SPIR-V Backend
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: level_zero:gpu
+            tests_selector: cts
+            extra_lit_opts: --param spirv-backend=True --param gpu-intel-gen12=True
```

Comment:
I have no objections to that conceptually, but I have a concern that we may not have enough HW available to finish that testing in reasonable time,

Reply:
Yes, I agree. I will keep the PR draft until we have completed discussing changes and potential consequences.

Reply:
yeah especially since we're using gen12 which takes forever to run as it is. can you bring this up at our next gatekeeper/CI meeting? there is one today but idk if there is time for this issue. will fwd.

Reply:
Sure, thank you